test.js

```diff
@@ -690,6 +690,27 @@ test('uses the custom successMessage callback if passed in as an option', functi
   })
 })
 
+test('custom successMessage has optional error parameter', function (t) {
+  var dest = split(JSON.parse)
+  var customErrorMessage = 'Custom response message'
+  var logger = pinoHttp({
+    customSuccessMessage: function (res, err) {
+      var optionalErrorMessage = err ? err.toString() : 'error is undefined'
+      return customErrorMessage + ' ' + optionalErrorMessage
```

Comment:
We should verify that the `err` object is present. Otherwise the test is invalid:

```suggestion
      t.ok(err)
      var optionalErrorMessage = err.toString()
      return customErrorMessage + ' ' + optionalErrorMessage
```